Buildings/Applications/DHC/CentralPlants/Cooling/Subsystems/CoolingTowerParellel.mo

```diff
@@ -0,0 +1,281 @@
+model CoolingTowerParellel
+  "Multiple identical cooling towers in parallel connection"
+  extends
+    Buildings.Applications.DataCenters.ChillerCooled.Equipment.BaseClasses.SignalFilter(
+     riseTimeValve=30,
+     use_inputFilter=true,
+    final numFil=num);
+
+  parameter Integer num(final min=1)=2 "Number of cooling towers";
+  replaceable package Medium=Buildings.Media.Water
+    "Condenser water medium";
+  parameter Modelica.Fluid.Types.Dynamics energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial
+    "Type of energy balance: dynamic (3 initialization options) or steady state"
+    annotation(Evaluate=true, Dialog(tab = "Dynamics", group="Equations"));
+  parameter Boolean show_T = true
+    "= true, if actual temperature at port is computed"
+    annotation(Dialog(tab="Advanced",group="Diagnostics"));
+  parameter Modelica.SIunits.MassFlowRate m_flow_nominal
+    "Nominal mass flow rate of condenser water in each tower"
+    annotation (Dialog(group="Nominal condition"));
+  parameter Modelica.SIunits.Pressure dp_nominal
+    "Nominal pressure difference of the tower"
+    annotation (Dialog(group="Nominal condition"));
+  parameter Real ratWatAir_nominal(final min=0,final unit="1")=0.625
+    "Design water-to-air ratio"
+    annotation (Dialog(group="Nominal condition"));
+  parameter Modelica.SIunits.Temperature TAirInWB_nominal
+    "Nominal outdoor (air inlet) wetbulb temperature"
+    annotation (Dialog(group="Heat transfer"));
+  parameter Modelica.SIunits.Temperature TWatIn_nominal
+    "Nominal water inlet temperature"
+    annotation (Dialog(group="Heat transfer"));
+  parameter Modelica.SIunits.TemperatureDifference dT_nominal
+    "Temperature difference between inlet and outlet of the tower"
+     annotation (Dialog(group="Heat transfer"));
+  parameter Modelica.SIunits.Power PFan_nominal
+    "Fan power"
+    annotation (Dialog(group="Fan"));
+  Medium.ThermodynamicState sta_a=
+      Medium.setState_phX(port_a.p,
+                          noEvent(actualStream(port_a.h_outflow)),
+                          noEvent(actualStream(port_a.Xi_outflow))) if
+         show_T "Medium properties in port_a";
+
+  Medium.ThermodynamicState sta_b=
+      Medium.setState_phX(port_b.p,
+                          noEvent(actualStream(port_b.h_outflow)),
+                          noEvent(actualStream(port_b.Xi_outflow))) if
+          show_T "Medium properties in port_b";
+
+  Modelica.Fluid.Interfaces.FluidPort_a port_a(
+    redeclare final package Medium=Medium)
+    "Fluid connector a (positive design flow direction is from port_a to port_b)"
+    annotation (Placement(transformation(extent={{-110,-10},{-90,10}})));
+  Modelica.Fluid.Interfaces.FluidPort_b port_b(
+    redeclare final package Medium=Medium)
+    "Fluid connector b (positive design flow direction is from port_a to port_b)"
+    annotation (Placement(transformation(extent={{90,-10},{110,10}})));
+  Modelica.Blocks.Interfaces.RealInput on[num](
```

Comment:
I noticed that the `on` signal was represented by `RealInput` quite often in your development, rather than `BooleanInput`. But I think it is more intuitive to represent an `on signal` with `BooleanInput`.

Reply:
Thanks for your comment. I've changed the `on` signals in both cooling tower models into `BooleanInput`.